pyomo/contrib/pyros/util.py

```diff
@@ -266,19 +266,53 @@ def add_bounds_for_uncertain_parameters(model, config):
 
 
 def transform_to_standard_form(model):
-    '''
-    Make all inequality constraints of the form g(x) <= 0
-    :param model: the optimization model
-    :return: void
-    '''
-    for constraint in model.component_data_objects(Constraint, descend_into=True, active=True):
-        if not constraint.equality:
-            if constraint.lower is not None:
-                temp = constraint
-                model.del_component(constraint)
-                model.add_component(temp.name, Constraint(expr= - (temp.body) + (temp.lower) <= 0 ))
+    """
+    Recast all model inequality constraints of the form `a <= g(v)` (`<= b`)
+    to the 'standard' form `a - g(v) <= 0` (and `g(v) - b <= 0`),
+    in which `v` denotes all model variables and `a` and `b` are
+    contingent on model parameters.
 
-    return
+    Parameters
+    ----------
+    model : ConcreteModel
+        The model to search for constraints. This will descend into all
+        active Blocks and sub-Blocks as well.
+
+    Note
+    ----
+    If `a` and `b` are identical and the constraint is not classified as an
+    equality (i.e. the `equality` attribute of the constraint object
+    is `False`), then the constraint is recast to the equality `g(v) == a`.
+    """
+    cons = [con for con in model.component_data_objects(Constraint,
+                                                        descend_into=True,
+                                                        active=True)].copy()
```

Comment:
Why do you need `.copy()` here?

Reply:
Good catch, @blnicho.  This should only be:
```python
cons = list(model.component_data_objects(Constraint, descend_into=True, active=True))
```

Reply:
(fixed with a note why we need to create the intermediate list)